package.json

```diff
@@ -36,8 +36,8 @@
     "karma-mocha": "^0.1.9",
     "karma-phantomjs-launcher": "^0.1.4",
     "mocha": "^1.21.4",
-    "react": "^0.14.0 || ^15.0.0",
-    "react-dom": "^0.14.0 || ^15.0.0",
+    "react": "^15.5.4",
```

Comment:
Do we need to update deps in this PR? Unsure of knock on effects. Maybe best left to keep the scope of this to just the code change.